Tasks/AzureContainerAppsV0/task.json

```diff
@@ -0,0 +1,170 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "35f6376b-4761-4488-a921-f882b2c994bb",
+    "name": "AzureContainerApps",
+    "friendlyName": "Azure Container Apps Deploy",
+    "description": "An Azure DevOps Task to build and deploy Azure Container Apps.",
+    "author": "Microsoft Corporation",
+    "helpMarkDown": "[Learn more about this task](http://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureContainerAppsV0/README.md)",
+    "releaseNotes": "Released new AzureContainerApps task for building and deploying Azure Container Apps.",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "minimumAgentVersion": "2.0.0",
```

Comment:
Please bump this version to `2.144.0` according to the [guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode10.md)

Reply:
Thanks for the tip -- if I'm also adding support for the node 16 handler, is `2.144.0` sufficient, or is there a minimum version I should be using that works for both the node 10 and 16 handlers? The `migrateNode16.md` document doesn't mention anything about the `minimumAgentVersion`, but just want to double check.

Reply:
Yes, `2.144.0` minimumAgentVersion will be for both node 10 and node 16 migrations. 
This is because the logic is: If the agent version between 2.144.0  and `2.206.1`, then the node 10 handler will be used, otherwise node 16.